### PR TITLE
Add docs about how to mount S3 buckets to use it with the docker

### LINKED ISSUE #1037: Issue: ImportError: cannot import name 'Literal' from 'pyparsing' (unknown location)
Getting it while importing lightly.loss and lightly.data

### PR DESCRIPTION
**Pull request by IgorSusmelj**
### Changes:

closes #1037

- Add instructions about how to mount an S3 bucket

[Load data directly from S3 buckets using s3fs-fuse — lightly 1.0.0 documentation.pdf](https://github.com/lightly-ai/lightly/files/7257811/Load.data.directly.from.S3.buckets.using.s3fs-fuse.lightly.1.0.0.documentation.pdf)
et

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ docs/source/tutorials/package/*
 docs/source/tutorials/platform/*
 docs/source/tutorials_source/platform/data
 docs/source/tutorials_source/platform/pizzas
+docs/source/tutorials_source/platform/__MACOSX
```
Comment (philippmwirth) [inline .gitignore:10]: what is this?

File: docs/source/docker/integration/s3fs-fuse.rst [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+On `Debian 9 or newer` or `Ubuntu 16.04 or newer` we can use the following terminal instructions to install it:
+
+.. code-block:: console
+
+    sudo apt install s3fs
+
+
+Below we show the output for installing s3fs on a Google Cloud Compute instance.
+
+.. code-block:: console
```
Comment (philippmwirth) [inline docs/source/docker/integration/s3fs-fuse.rst:65]: Is this really necessary?

File: docs/source/docker/integration/s3fs-fuse.rst [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+    docker run --gpus all --rm -it \
+        -v /s3-mount:/home/input_dir:ro \
+        -v /docker/output:/home/output_dir \
+        lightly/sampling:latest \
+        token=MYAWESOMETOKEN
+
+You can do the same for the docker output directory (in this example I used 
+`/docker/output`). Using this approach the pdf report as well as all output files
```
Comment (philippmwirth) [inline docs/source/docker/integration/s3fs-fuse.rst:169]: Can you make it a bit clearer that this happens only if you set the docker output directory to `/s3-mount`

File: .gitignore (hunk 1 above)
Comment (IgorSusmelj) [reply .gitignore:10]: This is a folder with thumbnails from OSX. We already have a line for another folder. I'll combine the two.

File: docs/source/docker/integration/s3fs-fuse.rst (hunk 2 above)
Comment (IgorSusmelj) [reply docs/source/docker/integration/s3fs-fuse.rst:65]: I think it's useful to see what one would expect to see.

File: docs/source/docker/integration/s3fs-fuse.rst (hunk 3 above)
Comment (IgorSusmelj) [reply docs/source/docker/integration/s3fs-fuse.rst:169]: I'll add some more context here. Thanks for the feedback!

File: docs/source/docker/integration/s3fs-fuse.rst [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+        -v /s3-mount:/home/input_dir:ro \
+        -v /docker/output:/home/output_dir \
+        lightly/sampling:latest \
+        token=MYAWESOMETOKEN
+
+You can do the same for the docker output directory (in this example I used 
+`/docker/output`). We can either use the same bucket and work on subfolders
+or use another bucket and repeat the procedure.
+
+If you use a mounted S3 bucket for the docker output can be very handy.
```
Comment (philippmwirth) [inline docs/source/docker/integration/s3fs-fuse.rst:172]: I think it should be "Using a mounted S3 bucket for the docker output can be very handy."

**Review by philippmwirth — APPROVED** [review]

**Pull request merged**